examples/functions/location/LIR_01_response_simple_StopPoint.xml

```diff
@@ -38,6 +38,21 @@
 							<PtMode>bus</PtMode>
 							<siri:BusSubmode>localBusService</siri:BusSubmode>
 						</Mode>
+						<Attribute>
+							<Text>
```

Comment:
This double <Text> is correct?

Reply:
it is correct. but I changed it now with the latest commit